localstack/services/ses/provider.py

```diff
@@ -0,0 +1,229 @@
+    ) -> ListTemplatesResponse:
+        for template in ses_backend.list_templates():
+            if isinstance(template["Timestamp"], (date, datetime)):
+                template["Timestamp"] = timestamp_millis(template["Timestamp"])
+        return call_moto(context)
+
+    @handler("DeleteTemplate")
+    def delete_template(
+        self, context: RequestContext, template_name: TemplateName
+    ) -> DeleteTemplateResponse:
+        if template_name in ses_backend.templates:
+            del ses_backend.templates[template_name]
+        return DeleteTemplateResponse()
+
+    @handler("GetIdentityVerificationAttributes")
+    def get_identity_verification_attributes(
+        self, context: RequestContext, identities: IdentityList
+    ) -> GetIdentityVerificationAttributesResponse:
+        attributes: VerificationAttributes = {}
+
+        for identity in identities:
+            if "@" in identity:
+                attributes[identity] = IdentityVerificationAttributes(
+                    VerificationStatus=VerificationStatus.Success,
+                )
+            else:
+                attributes[identity] = IdentityVerificationAttributes(
+                    VerificationStatus=VerificationStatus.Success,
+                    VerificationToken=long_uid(),
+                )
+
+        return GetIdentityVerificationAttributesResponse(
+            VerificationAttributes=attributes,
+        )
+
+    @handler("SendEmail")
+    def send_email(
+        self,
+        context: RequestContext,
+        source: Address,
+        destination: Destination,
+        message: Message,
+        reply_to_addresses: AddressList = None,
+        return_path: Address = None,
+        source_arn: AmazonResourceName = None,
+        return_path_arn: AmazonResourceName = None,
+        tags: MessageTagList = None,
+        configuration_set_name: ConfigurationSetName = None,
+    ) -> SendEmailResponse:
+        response = call_moto(context)
+
+        save_for_retrospection(
+            response["MessageId"],
+            context.region,
+            Source=source,
+            Destination=destination,
+            Subject=message["Subject"].get("Data"),
+            Body=message["Body"].get("Text", {}).get("Data"),
+        )
+
+        return response
+
+    @handler("SendTemplatedEmail")
+    def send_templated_email(
+        self,
+        context: RequestContext,
+        source: Address,
+        destination: Destination,
+        template: TemplateName,
+        template_data: TemplateData,
+        reply_to_addresses: AddressList = None,
+        return_path: Address = None,
+        source_arn: AmazonResourceName = None,
+        return_path_arn: AmazonResourceName = None,
+        tags: MessageTagList = None,
+        configuration_set_name: ConfigurationSetName = None,
+        template_arn: AmazonResourceName = None,
+    ) -> SendTemplatedEmailResponse:
+        message = ses_backend.send_templated_email(
```

Comment:
nit: We could think about using the generic dispatcher `call_moto(context)` here, instead of calling the moto backend method directly. The former would probably be a bit more stable/resilient, e.g., in case there are upstream changes which introduce new parameters to the backend method.

Reply:
Good idea :heavy_check_mark: